README.md

```diff
@@ -170,6 +170,7 @@ Asynchronous concurrent distributed networking framework for PHP
 
 * [amphp/mysql](https://github.com/amphp/mysql) - An async mysql client built on [amp](#amphp)
 * [amphp/pgsql](https://github.com/amphp/pgsql) - A non-blocking PostgreSQL library built on [amp](#amphp)
+* [amphp/redis](https://github.com/amphp/redis) - An async redis client built on [amp](#amphp)
```

Comment:
Think is [already listed under Caching](https://github.com/elazar/asynchronous-php/blame/ca95cc5d900f848f55a8596c310977b1d4a41d61/README.md#L149). I think it would be a bit redundant to also list it under Databases. If it's moved, the other Redis clients should be moved as well.